pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -584,7 +597,7 @@ def _unify_tensor_axes_returnn_meta(*inputs: Tensor) -> Tuple[Tuple[Tensor, ...]
     out_shape.append(dim_for_axis)
     for idx in broadcast_inputs_for_axis:
       broadcast_axes[idx].add(neg_torch_axis)
-  assert len(set(out_shape)) == len(out_shape) == max_ndim
+  assert len(set(out_shape)) == len(out_shape) == max_ndim - len(exclude_axes)
```

Comment:
I think this change is wrong. Just leave it as it was before?

Reply:
Ah sorry, now I see. But anyway, I think we should leave this logic, and rather make sure that `out_shape` is complete.